README.md

```diff
@@ -355,6 +355,19 @@ Mode for config_file.
 
 - *Default*: '0640'
 
+config_file_contents
```

Comment:
The word contents suggests the contents of the file, such as providing all of the contents, when this is actually an array of lines. For clarity, could this be changed to `config_file_lines` ?